OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListFragment.kt

```diff
@@ -283,11 +239,28 @@ class MyCoursesListFragment : OfflineSupportBaseFragment(), RefreshListener {
         refreshOnResume = true
     }
 
+    @Subscribe
+    fun onEventMainThread(event: IAPFlowEvent) {
+        if (!this.isVisible) {
```

Comment:
The `isVisible` and `isInForeground` always return `true`. Does it break the logic?